sample-app/src/components/InputDropdown.tsx

```diff
@@ -157,8 +160,8 @@ export default function InputDropdown({
   });
 
   function handleInputElementKeydown(evt: KeyboardEvent<HTMLInputElement>) {
-    if (evt.key === 'Enter') {
-      onInputChange(inputValue);
+    if (evt.key === 'Enter' && focusedSectionIndex === undefined && !onlySubmitOnOption) {
+      setLatestUserInput(inputValue);
       onSubmit(inputValue);
```

Comment:
instead of adding a new `onlySubmitOnOption` prop, would it be simpler/more flexible to pass `focusedSectionIndex` as a param to `onSubmit`?

Reply:
That could be used to prevent FilterSearch from submitting, but `setLatestUserInput` would still be called and the dropdown would be hidden when it shouldn't be. According to Rose, it should be apparent that nothing happens if the user presses Enter without selecting/focusing on a dropdown option